patches/chrome-browser-ui-commander-fuzzy_finder.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/commander/fuzzy_finder.cc b/chrome/browser/ui/commander/fuzzy_finder.cc
+index 5166bb5d04388..0149fbaea6b93 100644
+--- a/chrome/browser/ui/commander/fuzzy_finder.cc
++++ b/chrome/browser/ui/commander/fuzzy_finder.cc
+@@ -236,7 +236,7 @@ FuzzyFinder::~FuzzyFinder() = default;
+ double FuzzyFinder::Find(const std::u16string& haystack,
+                          std::vector<gfx::Range>* matched_ranges) {
+   matched_ranges->clear();
+-  if (needle_.size() == 0)
++  if (needle_.size() == 0 BRAVE_FUZZY_FINDER_NEEDLE_SIZE_CHECK)
```

Comment:
is this the right thing to do? Maybe we can do this only for `:>` prefix? Currently I don't think this is what we want:

https://user-images.githubusercontent.com/5928869/228268871-90d401ea-04c9-4877-a727-64fac415cfc0.mp4

Reply:
This is the correct behavior for the fuzzy finder but that's definitely a bug - I actually fixed this earlier but I removed the code which fixed it while cleaning up. I've added a test which should catch any regressions to it.

Reply:
One more thing: this change is not guarded with any feature and will be live for everyone after the PR is merged. Are you sure we want this?

wdyt about this?
```
double FuzzyFinder::Find(const std::u16string& haystack,
                          std::vector<gfx::Range>* matched_ranges) {
  matched_ranges->clear();
  BRAVE_FUZZY_FINDER_FIND
  if (needle_.size() == 0)
```
```
#define BRAVE_FUZZY_FINDER_FIND if (feature_enabled) return;
```

Reply:
@fallaciousreasoning this patch is not ideal because it modifies a line. It would be better to put `BRAVE_FUZZY_FINDER_NEEDLE_SIZE_CHECK` on the line above `return 0` and define it as `if (false)`